docs/dev/getting-started/core-concepts.md

```diff
@@ -72,7 +72,7 @@ Your web application's functionality can easily be extended by installing one or
 more of the many [available extensions][12] for Contao. A Contao extension is basically
 simply a Symfony bundle that you can load into your Symfony application - or will
 be automatically loaded in the Contao Managed Edition. See the
-[getting started article][13] on how to create an extension for Contao.
+[creating an extension article][13] on how to create an extension for Contao.
```

Comment:
I don't like this change, since it's an article in the getting starting section. There will probably be a more detailed article later on, hence the distinction.

Reply:
Feel free to revert that change then :)

Reply:
What do you think of this version?

```suggestion
article [getting started with creating an extension][13] on how to create an extension for Contao.
```

Reply:
"article __on__ getting started", but yes :)

Reply:
Please feel free to amend it